### PR TITLE
Remove Deprecated Junos Modules

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/remove_deprecated_modules.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
+  - Remove deprecated Junos Modules.
```
Comment (NilashishC) [inline changelogs/fragments/remove_deprecated_modules.yaml:3]: Can we list all the modules that were removed?

Comment (rohitthakur2590) [reply changelogs/fragments/remove_deprecated_modules.yaml]: Updated. Thanks

**Review by NilashishC — CHANGES_REQUESTED** [review]
We should also remove tests for these modules.

**Comment by rohitthakur2590** [comment]
> We should also remove tests for these modules.
removed!

**Pull request merged**